sdk/python/requirements.in

```diff
@@ -16,7 +16,7 @@ google-cloud-storage>=1.20.0,<2
 # NOTE: Maintainers, please do not require google-auth>=2.x.x
 # Until this issue is closed
 # https://github.com/googleapis/google-cloud-python/issues/10566
-kfp-pipeline-spec>=0.1.13,<0.2.0
+kfp-pipeline-spec>=0.1.14,<0.2.0
```

Comment:
This change itself doesn't have a hard dependency on the new version, but it's related.